install/libvips.js

```diff
@@ -80,9 +80,14 @@ try {
         throw new Error(`Use with glibc ${detectLibc.version} requires manual installation of libvips >= ${minimumLibvipsVersion}`);
       }
     }
-    if (!semver.satisfies(process.versions.node, process.env.npm_package_engines_node)) {
-      throw new Error(`Expected Node.js version ${process.env.npm_package_engines_node} but found ${process.versions.node}`);
+    const supportedNodeVersion = process.env.npm_package_engines_node || require('../package.json').engines.node;
+    if (!supportedNodeVersion) {
```

Comment:
Can the code inside this `if` block ever be reached given `require` will throw an error if there's a problem?

Reply:
Now that you mention it, `engines` would have to be defined but not `engines.node`; that probably wouldn't happen by accident. Happy to change it if you don't think it's worth the extra check + error message.

Reply:
Yes please to removing this check.

Reply:
:+1: